Web/Edubase.Services/IntegrationEndPoints/AzureMaps/AzureMapsService.cs

```diff
@@ -22,15 +22,17 @@ public class AzureMapsService : IAzureMapsService
             BaseAddress = new Uri("https://atlas.microsoft.com")
         };
 
-        private static readonly Policy RetryPolicy = Policy
+        private static readonly Policy RetryPolicy = CreateRetryPolicy();
+
+        private static Policy CreateRetryPolicy()
+        {
+            var retryIntervalSettings = ConfigurationManager.AppSettings["AzureMapService_RetryIntervals"].Split(',');
```

Comment:
I had assumed we planned to remove the retries not just add them to the config?

Reply:
I mentioned the retries are being looked at separately on ticket: 184252